metron-deployment/roles/metron-builder/tasks/main.yml

```diff
@@ -15,4 +15,7 @@
 #  limitations under the License.
 #
 ---
-hdp_repo_def: "http://public-repo-1.hortonworks.com/HDP/centos6/2.x/updates/2.5.0.0/hdp.repo"
+- name: Build Deployment Artifacts
+  local_action: shell cd {{ metron_build_dir }} && mvn clean package -DskipTests -P HDP-2.5.0.0,mpack,build-rpms
```

Comment:
I think I'd rather check in Maven and fail better earlier. What would you think about handling that as a standalone Jira?